packages/cozy-client/src/cli/index.spec.js

```diff
@@ -55,7 +56,7 @@ describe('createClientInteractive', () => {
     OAuthClient.prototype.fetchAccessToken.mockRestore()
   })
 
-  const setup = () => {
+  const setup = (withCustomFilePath = false) => {
```

Comment:
I think it would more extensible if instead of `withCustomFilePath`, we'd use an option object with `getSavedCredentials`.

```
   const setup = ({ getSavedCredentials } = {}) => { 
```

This way, in the test for custom file path, we can have

```
setup({ getSavedCredentials: () => 'custom file path' })
```

which would be more understandable IMHO. It lets define the mocked file path in the same
function where we `expect` it which is a plus.

Finally, we do not have to have a new "concept" `withCustomFilePath`.